HCTextField/ViewController.swift

```diff
@@ -19,16 +19,31 @@ class ViewController: UIViewController, HCTextFieldDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        configTextFields()
+
+        textFieldManager = HCTextFieldManager(textFields: [nameField, emailField, passwordField])
+        textFieldManager?.delegate = self
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+
+    // MARK: - Private implementations
+
+    private func configTextFields() {
         let checkmark = UIImage(named: "checkmark")
 
-        nameField.config(checkType: .notEmpty,
+        nameField.config(checkType: [.notEmpty, .noSpecialChar],
                          placeholder: "Full name",
+                         specialChars: ["/", "&", "*", "$"],
```

Comment:
let's put the special character set inside of the HCTextField, don't need to let the user set it.
Please include space " " too.